master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
```

Comment:
There are some examples at the end.  The reference docs for the Node resource already have a sample node yaml anyway.